source/layouts/blog.erb

```diff
@@ -18,23 +18,115 @@
     <![endif]-->
   </head>
 
-  <body class="<%= page_classes %>">
-    <div class="site-header">
+    <body class="<%= page_classes %>">
+    <header class="site-header">
       <div class="site-header__brand">
         <a class="site-header__link" href="https://www.unboxedconsulting.com/">
-          Unboxed
+        Unboxed
         </a>
       </div>
+    </header>
+
+    <div class="blog-article">
+      <div class="blog-article__container">
+
+        <section class="blog-header">
+          <h1 class="blog-header__title">
+            <%= current_page.data.title %>
+          </h1>
+          <a class="blog-header__author" href="">
+            <%= current_page.data.author %>
+          </a>
+          <p class="blog-header__published-date">
+            <span>
+              <%= Time.parse(current_page.data.date).strftime("%B %e, %Y") %>
+            </span>
+          </p>
+          <div class="blog-header__tags blog-header__tag-border">
+          <% data.tags.each do |tag| %>
+            <div class="blog-header__tag-link
+              <%= "blog-header__tag-active blog-header__tag-border-active" if current_article.tags.include?(tag) %>">
+                <a href=""><%= tag %></a>
```

Comment:
Good point! Changed here: 94ff141 together with border color, just have noticed it was too dark